src/NuGet.Core/NuGet.Common/Utility/NuGetLogCodeUtility.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.Common
+{
+    public static class NuGetLogCodeUtility
+    {
+        public static bool AreVulnerabilitiesInCodes(IEnumerable<NuGetLogCode> nugetLogCodes)
+        {
+            foreach (var code in nugetLogCodes)
+            {
+                if (IsVulnerabilityCode(code))
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        public static bool IsVulnerabilityCode(NuGetLogCode nuGetLogCode)
```

Comment:
I feel like the word `Code` is used too many times. Think about callers of the API: `NuGetLogCodeUtility.IsVulnerabilityCode(logMessage.Code);`

Is there something that would look nicer for an API customer?

```csharp
NuGetLogCodeUtility.IsPackageVulnerability(logMessage.Code)
NuGetLogCodeUtility.IsVulnerability(logMessage.Code)
logMessage.Code.IsVulnerability()
logMessage.Code.IsVulnerabilityCode()
logMessage.Code.IsPackageVulnerability()
logMessage.Code.IsVulnerablePackage()
```